i have updated the corrected images paths

khadijahMu opened the pull request.

khadijahMu committed:
images

khadijahMu committed:
images

khadijahMu committed:
styles

khadijahMu committed:
mobile view of hotel

khadijahMu committed:
hotel website

khadijahMu committed:
Hotel Website

khadijahMu committed:
Hotel website

khadijahMu committed:
Hotel website

khadijahMu committed:
Hotel-website

khadijahMu committed:
Hotel website

khadijahMu committed:
Hotel Website

khadijahMu committed:
hotel website

khadijahMu committed:
Hotel-web

khadijahMu committed:
Hotel web

khadijahMu committed:
initial commit

khadijahMu committed:
new commit

khadijahMu committed:
commit 1

khadijahMu committed:
new commit

khadijahMu committed:
new commit

khadijahMu committed:
new commit

khadijahMu committed:
new commit

khadijahMu committed:
new commit

khadijahMu committed:
new commit

khadijahMu committed:
"changes"

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated changes

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated images

khadijahMu committed:
updated imag

khadijahMu committed:
updated changes

khadijahMu committed:
images updated

khadijahMu committed:
Add index.html, styles.css, and main.js

khadijahMu committed:
updated images

khadijahMu committed:
new changes

khadijahMu committed:
Merge pull request #1 from khadijahMu/development

Update README.md

khadijahMu committed:
updated changes

khadijahMu committed:
Merge branch 'main' of https://github.com/khadijahMu/Hotel-website-mobile

Xfarouq commented:
@khadijahMu why are your files empty??

khadijahMu committed:
updated changes 2

khadijahMu commented:
let me check them

khadijahMu commented:
i have added my files

Devsalmasaeed reviewed (changes requested):
Hi @khadijahMu,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- All repetitive content of the webpage should be rendered dynamically using JavaScript (Why Choose Us, Special Offers, Reviews and Recent Posts)
- Deploy your website on render and update your README

 

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

khadijahMu committed:
dynamic render update

khadijahMu committed:
styles update

khadijahMu committed:
styles update

khadijahMu committed:
Update README.md

khadijahMu committed:
Merge branch 'main' of https://github.com/khadijahMu/Hotel-website-mobile

khadijahMu committed:
Update README.md

khadijahMu commented:
1. I have changed all repetitive contents and rendered them dynamically 
2. i have updated my Readme and redployed my site on render

Xfarouq reviewed (changes requested):
Hi @khadijahMu 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- It's best practice to always not leave spaces in your code, so please remove them and make sure your code is well indented
- put all your images folder in a folder named assests  for best practices
- Remove unused css files which isn't used for the code
- Make your menu bar hidden unless when the hamburger menu is clicked for it to pop up
- Remove your html codes for the sections which you rendered using javascript

- kindly consider having the app's name on the splash screen as requested in the project requirement or as given on the design

### Optional Suggestions
- You can make your review section to stop showing the horixontal bar by doing overflow-x: scroll

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

khadijahMu committed:
Delete hotel images directory

khadijahMu committed:
Delete images

khadijahMu committed:
Create assets

khadijahMu committed:
Delete assets

khadijahMu committed:
Create assets

khadijahMu committed:
Add files via upload

khadijahMu committed:
Delete photos directory

khadijahMu committed:
Delete stylesss.css

khadijahMu committed:
latest changes

khadijahMu committed:
Merge branch 'main' of https://github.com/khadijahMu/Hotel-website-mobile

khadijahMu commented:
I have made all the required changes

Xfarouq commented:
Ok you can submit again for review

On Sat, Nov 2, 2024, 16:18 khadijahMu ***@***.***> wrote:

> I have made all the required changes
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/khadijahMu/Hotel-website-mobile/pull/2#issuecomment-2453020814>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3SWBV4PL7DQSZGVTEMA25DZ6TULNAVCNFSM6AAAAABQXUKRN6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINJTGAZDAOBRGQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

Xfarouq commented on main.js:
```diff
@@ -1,44 +1,226 @@
-        additionalReviews.style.display = 'none';  // Hide the reviews
-    }
-});
-
```
avoid leaving whitespaces please

Xfarouq commented on main.js:
```diff
@@ -1,44 +1,226 @@
-closeMenu.addEventListener('click', function() {
-    menuPage.style.display = 'none';  // Hide the menu page
-});
-
```
avoid leaving spaces

Xfarouq commented on index.html:
```diff
@@ -1,726 +1,506 @@
-            </div>
-        </div>
-                <!-- fourth deal -->
-        <div class="special-offer">
```
Please remove all html used for the rendered sections just leave the ID name used to call the rendered sections in javascript

Xfarouq reviewed (changes requested):
Hi @khadijahMu 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- You still haven't implemented previous changes
- Your images should all be in one folder called images and then put the images folder in assets folder
- If your having any difficulty or you don't understand anything please your always free to reach out and ask
- kindly consider having the app's name on the splash screen as requested in the project requirement or as given on the design

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

khadijahMu committed:
Create images

khadijahMu committed:
Delete assets directory

khadijahMu committed:
Create images

khadijahMu committed:
Delete assets directory

khadijahMu committed:
Create images

khadijahMu committed:
Delete assets directory

khadijahMu committed:
Create images

khadijahMu committed:
Add files via upload

khadijahMu committed:
Delete assets/images directory

khadijahMu committed:
Create images

khadijahMu committed:
Delete assets/images directory

khadijahMu committed:
Create assets

khadijahMu committed:
Delete images directory

khadijahMu committed:
Create images

khadijahMu committed:
Delete assets directory

khadijahMu committed:
Create images

khadijahMu committed:
Add files via upload

khadijahMu commented:
1- i have added the project name as splash screen
2- my last deal section was not rendered dynamically, but i have deleted those that were dynamically rendered (review,why choose us, and special offers).
3- i have created the assets folder.
4-i have removed lines and white spaces.

khadijahMu committed:
update 2

khadijahMu committed:
Merge branch 'main' of https://github.com/khadijahMu/Hotel-website-mobile

khadijahMu committed:
final

khadijahMu committed:
Delete photos directory

khadijahMu committed:
Delete hotel images directory

khadijahMu committed:
Delete images

khadijahMu committed:
commit 2

khadijahMu committed:
Merge branch 'main' of https://github.com/khadijahMu/Hotel-website-mobile
